apache-http-client/src/main/java/com/joyent/http/signature/apache/httpclient/HttpSignatureAuthScheme.java

```diff
@@ -202,18 +204,10 @@ protected Header signRequestHeader(final Credentials credentials,
             request.setHeader(HttpHeaders.DATE, stringDate);
         }
 
-        final String authz;
-
-        try {
-            authz = signer.get().createAuthorizationHeader(
-                    login, fingerprint, keyPair, stringDate);
-        } catch (HttpSignatureException e) {
-            String details = String.format("Unable to authenticate [%s] with "
-                    + "fingerprint [%s] using keypair [%s]",
-                    login, fingerprint, keyPair);
-            throw new AuthenticationException(details, e);
-        }
+        signatureCacheMap.computeIfAbsent(credentials, NEW_CACHE_FUNCTION);
```

Comment:
If we cache for each duplicate date value, would their be any advantage to turning it off? Why would you want to?